toolkits/global/packages/global-corporate-footer/demo/context.json

```diff
@@ -1,7 +1,65 @@
 {
```

Comment:
I think there should be a "Configuration" section in the README that shows how the data should be structured - because you don't actually need the `default: {}` and `extended: {}` wrappers if you are not trying to display two different types of demo. So...

```
{
  "navigation": {},
  "image": {},
  "currentYear": ''
}
```

Then explain in that section why you have the wrappers in `demo/context.json`